$NetBSD: patch-aa,v 1.6 2010/02/08 01:47:14 jakllsch Exp $

- Fix the SUBDIRS stuff so it actually works right with multiple automake
  versions.

- Use supported automake methods to install the example config file to
  egconfdir instead of hacking the configure script and Makefile.in directly.


--- Makefile.am.orig	2010-01-19 10:39:11.000000000 +0000
+++ Makefile.am
@@ -153,7 +153,8 @@ avrdude_SOURCES = \
 
 man_MANS = avrdude.1
 
-sysconf_DATA = avrdude.conf
+egconfdir=	${datadir}/examples/avrdude
+egconf_DATA = avrdude.conf
 
 install-exec-local: backup-avrdude-conf
 
@@ -162,8 +163,6 @@ distclean-local:
 
 # This will get run before the config file is installed.
 backup-avrdude-conf:
-	@echo "Backing up avrdude.conf in ${DESTDIR}${sysconfdir}"
-	@if test -e ${DESTDIR}${sysconfdir}/avrdude.conf; then \
-		cp -pR ${DESTDIR}${sysconfdir}/avrdude.conf \
-			${DESTDIR}${sysconfdir}/avrdude.conf.bak; \
-	fi
+	@echo "No back up of avrdude.conf in ${DESTDIR}${sysconfdir} is required"
+	@echo "because we simply install an example into ${DESTDIR}${egconfdir}"
+	@echo "and leave it to the user to install/update the actual working copy"
